setup/db/db/schema-4910to4920.sql

```diff
@@ -236,3 +236,5 @@ CALL `cloud`.`IDEMPOTENT_INSERT_GUESTOS_HYPERVISOR_MAPPING`('Xenserver', '7.0.0'
 CALL `cloud`.`IDEMPOTENT_INSERT_GUESTOS_HYPERVISOR_MAPPING`('Xenserver', '7.0.0', 'Ubuntu Trusty Tahr 14.04', 256, 0);
 
 DROP PROCEDURE `cloud`.`IDEMPOTENT_INSERT_GUESTOS_HYPERVISOR_MAPPING`
+
```

Comment:
@syed 4.9.2.0 is already released, this need to go into 4.9.2.0->4.9.3.0 upgrade path.